.travis.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     - name: "Change detection"
       if: type = pull_request
       script: ./detect-pr-changes.sh
+    - name: "Spanner Emulator"
+      script: ./spanner-ITs-against-emulator.sh;
```

Comment:
I will probably noodle on this filename further - I wonder whether putting it under apis/Google.Cloud.Spanner.Data would make sense. But don't worry about that for this PR.
(We'll also want to work out a way of only doing this when something about Spanner has changed, rather than making every PR take 15 minutes to run, but I can do that.)